test/benchmark_test.js

```diff
@@ -1,18 +1,39 @@
-var cryptohat = require("..");
-var hat = require("hat");
+(function() {
+
+var cryptohat = __test.cryptohat;
+var hat = __test.hat;
 
 // Runs the target function for a number of iterations and returns the number
 // of nanoseconds passed per iteration.
-var benchmark = function (iterations, code) {
-  var t0 = process.hrtime();
-  for (var i = 0; i < iterations; ++i) {
-    code();
+var benchmark = null;
+
+if (typeof(process) !== "undefined" && typeof(process.hrtime) === "function") {
```

Comment:
I can use this pattern to simplify the CSPRNG engine detection though. Thank you!